src/ui/script_ui.cpp

```diff
@@ -221,9 +273,20 @@ static int CclSetVideoResolution(lua_State *l)
 }
 
 /**
+** <b>Description</b>
+**
 **  Get the video resolution.
 **
 **  @param l  Lua state.
+**
+** Example:
+**
+** <div class="example">
+** 	<code>
+**		resolution = GetVideoResolution()
```

Comment:
here maybe this example would be better:
```
width,height = GetVideoResolution()
print("Resolution  is " .. width .. "x" .. height)
```

Reply:
Fixed the GetVideoResolution example
https://github.com/Wargus/stratagus/commit/72f4bce9a18b577169f05e468e99e81cce150a7d#r52423776

Reply:
Fixed the GetVideoResolution example
https://github.com/Wargus/stratagus/commit/72f4bce9a18b577169f05e468e99e81cce150a7d#r52423776